release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -122,6 +122,17 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+==== Run a pod in debug mode
```

Comment:
Maybe use a gerund here:

"Running a pod in debug mode"

or alternatively something like "Pod debug terminals"

Reply:
Heading needs an ID/anchor here.